src/nvim/option.c

```diff
@@ -598,23 +598,20 @@ void set_init_1(void)
     }
   }
 
-  /*
-   * 'maxmemtot' and 'maxmem' may have to be adjusted for available memory
-   */
+  // 'maxmemtot' and 'maxmem' are ignored in Neovim, but we set them to the
+  // amount of available memory.
```

Comment:
If we're not removing it entirely, shouldn't  the option be made "hidden" (as we did with `termencoding`), by removing the `varname` line in `options.lua` ?